src/strategies/saddle-finance/index.ts

```diff
@@ -49,25 +50,24 @@ export async function strategy(
     { blockTag }
   );
 
-  const vestingAddrsBalanceRes = multicall(
+  const beneficiaries = multicall(
     network,
     provider,
     abi,
     vestingContractAddrs.map((vestingContractAddress: any) => [
-      SDLTokenAddress,
-      'balanceOf',
-      [vestingContractAddress.toLowerCase()]
+      vestingContractAddress.toLowerCase(),
+      'beneficiary'
     ]),
     { blockTag }
   );
 
-  const beneficiaries = multicall(
+  const vestedAndUnclaimedAmountRes = multicall(
     network,
     provider,
     abi,
     vestingContractAddrs.map((vestingContractAddress: any) => [
       vestingContractAddress.toLowerCase(),
-      'beneficiary'
+      'vestedAmount'
```

Comment:
The logic here would need to be to get both the `vestedAmount` as well as `released` from the same contract, and use `walletBalance + vestedAmount - released` as the voting amount

Reply:
I was incorrect about this logic ^